rdrf/rdrf/templates/rdrf_cdes/alert_language_is_not_supported.html

```diff
@@ -1,30 +1,34 @@
 {% load i18n %}
 {% load get_language_settings_codes %}
+{% load get_user_requested_language %}
+{% get_user_requested_language as USER_REQUESTED_LANGUAGE %}
 {% get_language_settings_codes as LANGUAGES %}
 
 <script type="text/javascript">
     $(document).ready(function(){
-        const browser_language = navigator.language;
-        const supported_languages = {{ LANGUAGES|safe }};
-        const is_browser_language_in_settings = [browser_language, browser_language.split('-')[0]].some((language) => supported_languages.includes(language));
+        const user_requested_language = "{{ USER_REQUESTED_LANGUAGE }}";
+        const supported_languages = {{ LANGUAGES|safe }}.map(language => language.toLowerCase());
+        const is_user_requested_language_in_settings = [
+            user_requested_language.toLowerCase(), user_requested_language.split('-')[0]
+        ].some((language) => supported_languages.includes(language));
```

Comment:
Can probably change this to do a "find", so that it returns at the first instance of a match.